lang/cs/Org.Apache.REEF.IMRU.Examples/Run.cs

```diff
@@ -36,24 +38,27 @@ public class Run
     {
         private static readonly Logger Logger = Logger.GetLogger(typeof(Run));
 
-        private static void RunMapperTest(IConfiguration tcpPortConfig, bool runOnYarn, int numNodes)
+        private static void RunMapperTest(IConfiguration tcpPortConfig, bool runOnYarn, int numNodes, string filename)
         {
             IInjector injector;
+            IConfiguration fileSystemConfig;
 
             if (!runOnYarn)
             {
                 injector =
                     TangFactory.GetTang()
                         .NewInjector(OnREEFIMRURunTimeConfiguration<int, int, int>.GetLocalIMRUConfiguration(numNodes), tcpPortConfig);
+                fileSystemConfig = LocalFileSystemConfiguration.ConfigurationModule.Build();
             }
             else
             {
                 injector = TangFactory.GetTang()
                     .NewInjector(OnREEFIMRURunTimeConfiguration<int, int, int>.GetYarnIMRUConfiguration(), tcpPortConfig);
+                fileSystemConfig = HadoopFileSystemConfiguration.ConfigurationModule.Build();
             }
 
             var mapperCountExample = injector.GetInstance<MapperCount.MapperCount>();
-            mapperCountExample.Run(numNodes - 1);
+            mapperCountExample.Run(numNodes - 1, filename, fileSystemConfig);
```

Comment:
Similar as above, fileSystemConfig should be passed when creating the injector above. And remove it from Run signature.